### PR TITLE
Docker docs for RP 5

### PR DESCRIPTION
**Pull request by AlehB**

### CODE REVIEW

File: src/md/src/InstallationSteps/DeployReportPortal.md [hunk 1]
```diff
@@ -26,11 +26,18 @@ Where:
...
+
+You can get the host IP address by using the following docker command:
+
+```shell
+ $ docker-machine ip default
+```
 
   ```
-  http://IP_ADDRESS:8080
+  http://IP_ADDRESS:9090
```
Comment (avarabyeu) [inline src/md/src/InstallationSteps/DeployReportPortal.md]: Guys, please, do not change default IP. 
Users got used to previous one

File: src/md/src/InstallationSteps/SetupSSL.md [hunk 2]
```diff
@@ -1,139 +1,53 @@
...
-```
-After the conversion is completed, you need to add the converted certificate to the jks container.
-To do it, open up the bin folder in your JAVA JDK directory and run:
-```$xslt
-keytool -import -alias your-alias -keystore certstore.jks -file cert.der
-```
-NB! Remember your password, after adding it to container.<br>
-Add the certstore.jks file to resources folder in the classpath of your project.
+gateway:
+    image: traefik:1.6
```
Comment (avarabyeu) [inline src/md/src/InstallationSteps/SetupSSL.md]: 1.7?

Comment (DzmitryHumianiuk) [reply src/md/src/InstallationSteps/SetupSSL.md]: @AlehB ^

File: src/md/src/InstallationSteps/DeployReportPortal.md (hunk 1 above)
Comment (DzmitryHumianiuk) [reply src/md/src/InstallationSteps/DeployReportPortal.md]: @AlehB

File: src/md/src/InstallationSteps/SetupSSL.md (hunk 2 above)
Comment (AlehB) [reply src/md/src/InstallationSteps/SetupSSL.md]: @avarabyeu , there is traefik:1.6 in the latest docker-compose.yml provided by @pbortnik

File: src/md/src/InstallationSteps/DeployReportPortal.md (hunk 1 above)
Comment (AlehB) [reply src/md/src/InstallationSteps/DeployReportPortal.md]: @avarabyeu , it was done according to the latest docker-compose.yml provided by @pbortnik
if we are going to keep 8080 and 8081 ports, let's make the same in the documentation

**Pull request merged**